index.js

```diff
@@ -25,6 +38,55 @@ function Changelog( options ) {
 	}
 }
 
+Changelog.ticketParsers = {
+	// Order of keys is important!
+	// When passed several ticketTypes, they will be re-ordered in the order of the keys of this map
+	// This works around github's (or other) regex to be greedy
+	jira: function ( commit, ticketUrl ) {
+		var tickets = [];
+
+		// Sane global exec with iteration over matches
+		commit.replace(
+			/Fix(?:e[sd])? ((?:([A-Z]{1,10})-?)[A-Z]+-\d+)/g,
```

Comment:
How did you come up with this regex? It doesn't seem to comply with the JIRA project key format.